internal/service/ec2/vpc_managed_prefix_list.go

```diff
@@ -182,9 +183,31 @@ func resourceManagedPrefixListRead(ctx context.Context, d *schema.ResourceData,
 func resourceManagedPrefixListUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	conn := meta.(*conns.AWSClient).EC2Conn
 
-	if d.HasChangesExcept("tags", "tags_all") {
+	id := d.Id()
```

Comment:
I am assuming the reason for pulling this into a local variable was to reduce memory?

Reply:
Yeah, I figured why call a function 10x times instead just ref a var